namer/namer.cc

```diff
@@ -2001,14 +2000,20 @@ class TreeSymbolizer {
         // Prevent a crash by validating the arguments before we do any symbol lookups or manipulations
         if (send->hasPosArgs() || send->hasKwArgs() || send->block() != nullptr) {
             if (send->numPosArgs() > 1) {
-                if (auto e = ctx.beginError(send->loc, core::errors::Namer::InvalidTypeDefinition)) {
-                    e.setHeader("Too many args in type definition");
+                auto extraArgsLoc = send->getPosArg(1).loc().join(send->getPosArg(send->numPosArgs() - 1).loc());
+                if (auto e = ctx.beginError(extraArgsLoc, core::errors::Namer::InvalidTypeDefinition)) {
+                    e.setHeader("Too many arguments in `{}` definition for `{}`", send->fun.show(ctx),
+                                typeName->cnst.show(ctx));
+                    auto firstArgLoc = send->getPosArg(0).loc();
+                    auto argsLoc = send->argsLoc();
+                    auto replacementRange = core::Loc(ctx.file, firstArgLoc.endPos(), argsLoc.endPos());
+                    e.replaceWith("Retain only the first argument", replacementRange, "",
```

Comment:
```suggestion
                    e.replaceWith("Retain only the first argument", replacementRange, "{}",
```

I think you need this or else you won't get any autocorrect output, and if that's not the case then I'm kind of surprised. I see that your autocorrect exp file has the intended output, and I'm not sure if that's because you programmatically generated it or you manually edited it.

I guess CI will tell us the answer.